src/main/groovy/de/dkfz/roddy/execution/jobs/cluster/pbs/PBSJobManager.groovy

```diff
@@ -343,19 +343,16 @@ class PBSJobManager extends ClusterJobManager<PBSCommand> {
                     String[] split = line.split(" ")
                     final int ID = getPositionOfJobID()
                     final int JOBSTATE = getPositionOfJobState()
-                    if (logger.isVerbosityHigh()) {
-                        System.out.println("QStat BEJob line: " + line)
-                        System.out.println("	Entry in arr[" + ID + "]: " + split[ID])
-                        System.out.println("    Entry in arr[" + JOBSTATE + "]: " + split[JOBSTATE])
-                    }
+                    logger.info("QStat BEJob line: " + line)
```

Comment:
logger.info(["QStat BEJob line: " + line, 	Entry in arr[" + ID + "]: " + split[ID] + ("    Entry in arr[" + JOBSTATE + "]: " + split[JOBSTATE]].join("\n")) ?